packages/hosted-form-v2/src/index.ts

```diff
@@ -0,0 +1,6 @@
+export { initializeHostedInput, notifyInitializeError } from './iframe-content';
```

Comment:
Can you provide an example of how other packages could consume this package?

Reply:
I think it is exported to be used in bigpay https://github.com/bigcommerce/bigpay/blob/1d0e8fc28ad1f17779662d8ab603d8ed04137b8e/app/views/payments/onsite/hosted_form_fields/show.html.erb#L2
https://github.com/bigcommerce/bigpay/blob/1d0e8fc28ad1f17779662d8ab603d8ed04137b8e/app/views/payments/onsite/hosted_form_fields/error.html.erb#L4

These exported functions are not used directly in checkout js, other than for e2e tests there to generate iframe 
https://github.com/bigcommerce/checkout-js/blob/918be1808209440f53b3053a8464abfbbe553260/packages/adyen-integration/e2e/support/hostedField.ejs#L10

Reply:
To clarify, I'm interest to know, once this package is done, how manual order can consume it to create hosted fields. And how it will eventually replace `hosted-form` v1 in the `core` package.

Reply:
Yes, that is still a open question that we need help with as mentioned in slack: `as to from where do we expose the HostedFormFieldFactory, do we create a new service in the  core package and export from there OR do we create a new HostedFormService in hosted-form-v2 and export it from core package because right now all we are able to use should be exported via core afaik?`

Reply:
I think the intention is to publish this as a standalone package so that manual orders UI can import it without bringing in all the unnecessary code of Checkout SDK. And eventually we can modify [My Account](https://github.com/bigcommerce/storefront-account-payments) and checkout to use the new implementation and remove the old one.

We can probably still achieve the same thing without creating a new monorepo package. But the monorepo setup can help enforce boundary. Having said that, we can defer the work if you feel it's quicker to modify the existing code in `core` instead.

No matter which approach we take, we still need to do some refactoring so that `HostedFormFieldFactory` doesn't depend on `CheckoutStore`: https://github.com/bigcommerce/checkout-sdk-js/blob/a6e895a8f1fe9a5e1dd849b345c800953d06a389/packages/core/src/hosted-form/hosted-form-factory.ts#L21 This is because it brings in a lot of other dependencies, which makes it hard to code split.